tests/unit_tests/test_wandb_artifacts_cache.py

```diff
@@ -155,27 +156,27 @@ def test_artifacts_cache_cleanup_empty():
     assert reclaimed_bytes == 0
 
 
-def test_artifacts_cache_cleanup():
-    cache_root = os.path.join("cache", "obj", "md5")
+def test_artifacts_cache_cleanup(tmp_path):
+    cache_root = os.path.join(tmp_path, "obj", "md5")
 
     path_1 = os.path.join(cache_root, "aa")
     os.makedirs(path_1)
     with open(os.path.join(path_1, "aardvark"), "w") as f:
         f.truncate(5000)
-    time.sleep(0.1)
+        f.flush()
```

Comment:
I honestly can't tell you why this was necessary, but on my local machine the `time.sleep()` calls were ineffective and it was frequently the case that the `st_atimes` for the files were out of order. This works for me, but if I was superstitious I would also use `os.fsync()`.